UnityProject/Assets/Scripts/Systems/Antagonists/Objective.cs

```diff
@@ -1,22 +1,67 @@
 using System;
-using System.Collections;
 using System.Collections.Generic;
 using Items;
 using Logs;
 using UnityEngine;
 
 namespace Antagonists
 {
+	[System.Serializable]
+	public class ObjectiveAttribute
+	{
+		public string name;
+		[HideInInspector]
+		public short index = -1;
+
+		public ObjectiveAttributeType type;
+		[HideInInspector]
```

Comment:
change these to ``[NonSerialized]`` instead or better yet, make them properties.

``public string PlayerID {get; set;}``